src/state_management.h

```diff
@@ -25,6 +26,8 @@ namespace ara { namespace sm {
         /** @brief Fills [SWS_SM_00021] */
         com::TriggerIn triggerIn;
         com::TriggerInOut triggerInOut;
+        SMStateType internalState = SMStateType::Off;
+        exec::StateClient * stateClient = nullptr;
```

Comment:
shall be 'exec::StateClient* stateClient' or 'exec::StateClient *stateClient'. Nowadays, I think the first option is common

Reply:
Done